README.md

```diff
@@ -17,8 +27,57 @@ For more information on Fixtures, check out this [pytest documentation](https://
 A **Mock** is an object that simulates the behavior of another object such as a class or function. They are used to simulate the behavior of real objects for testing purposes.
 For more information on Mocks, check out this [unittest documentation](https://docs.python.org/3/library/unittest.mock.html) on the mock module.
 
-**Example:**
+## Installation
+
+To use the **JupyterHub Pytest Plugin**, you will first need to install it using pip by either:
+
+- installing it locally
+  ```bash
+  pip install --upgrade pip
+  pip install git+https://github.com/jupyterhub/pytest-jupyterhub.git@main
+  ```
+- adding it to the `requirements.txt` file of your project:
+
+  ```python
+  # in requirements.txt
+  git+https://github.com/jupyterhub/pytest-jupyterhub.git@main
+  ```
```

Comment:
This is subject to how each project is managing their dependencies, and it's hard and not recommended to cover all cases, so let's remove and leave it to each project to define.